docs/sql-guide/functions/function-tuple.md

```diff
@@ -7,85 +7,39 @@ grand_parent: SQL guide
 
 # TUPLE() function
 
-The `TUPLE()` function takes in n arguments and returns a tuple containing an element for each argument.
+`TUPLE()` is a mathematical function that returns an ordered list of values corresponding to and derived from a supplied argument.
 
 ## Syntax
 
+```sql
+TUPLE (
+  [ordinal_position,...] |
+  [column_name,...]
+  )
 ```
+<!-- original syntax
 TUPLE(expr1,expr2,...)
-```
+-->
 
 ## Arguments
 
-| Argument | Data type | Description | Required | Further information |
-|---|---|---|---|---|
-| expr | any | an expression to represent an element in the resultant tuple. Supports 1 to n arguments of any type. | Yes | |
+| Argument | Description | Required | Further information |
+|---|---|---|---|
+| `ordinal_position` | An ordinal position defined in the `MAP` clause of a `BULK INSERT` statement | When used in BULK INSERT `TRANSFORM` clause | [BULK INSERT statement](/docs/sql-guide/statements/statement-insert-bulk) |
+| `column_name`| Table column name in existing table | When used in SELECT statement `select-list` | [SELECT statement](/docs/sql-guide/statements/statement-select) |
 
 ## Returns
 
 | Data type | Value |
 |---|---|---|
-| tuple | tuple with an element per evaluated expression `(eval_expr1,eval_expr2,...)` | 
-
-## Additional information
+| tuple | tuple with an element per evaluated expression `(eval_expr1,eval_expr2,...)` |
```

Comment:
I didn't understand what this means which is why it's still here.
The second table is the include file that's shared with multiple other pages